tests/test_standard_model.py

```diff
@@ -22,7 +21,7 @@
 
 
 @pytest.fixture(scope='function')
-def data_df_dict(transform_stage):
+def data_df_dict(guided_transform_stage):
```

Comment:
I know `_insert_unique_keys` is part of the base TransformStage class so everything still works with `guided_transform_stage._insert_unique_keys`, but it looks weird since standard model is part of the auto transformer.  Can we change all references in this file from `guided_transform_stage` to `auto_transform_stage`?